### ISSUE TITLE
[a11y] Document should have one main landmark (subplat)

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa#12919]
## Description

> Document should have one main landmark ([ref](https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome))

### Steps to reproduce

1. Log into FxA and go to https://payments-stage.fxa.nonprod.cloudops.mozgcp.net/subscriptions
2. Run axe DevTools

### Expected result

No a11y errors/warnings

### Actual result

> Fix the following:
> Document does not have a main landmark

Axe doesn't give a lot of advice here, but #12918 did have a proposal of wrapping the entire panel looking element in a `<main>` tag (or using `<main>` instead of one of the `<div>` tags), which would probably solve this (as well as the `<header>` issue).

### Environment

What browser and operating system are you using?

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5113)

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
